<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomDialog</class>
 <widget class="QDialog" name="RoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wdg_title" native="true">
     <property name="minimumSize">
      <size>
       <width>801</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lb_title">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>房间号 000000</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>497</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_min">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_max">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_close">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdg_mid" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wdg_left" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="VideoItem" name="wdg_video" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="IMToolBox" name="wdg_right" native="true">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>500</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wdg_bottom" native="true">
     <property name="minimumSize">
      <size>
       <width>801</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pb_openVideo">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>打开视频设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_openAudio">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>打开音频设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbx_tuzi">
        <property name="minimumSize">
         <size>
          <width>87</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>87</width>
          <height>30</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>无滤镜</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>兔耳朵</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>圣诞帽</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>238</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_quit">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>141</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>离开房间</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IMToolBox</class>
   <extends>QWidget</extends>
   <header location="global">IMToolBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoItem</class>
   <extends>QWidget</extends>
   <header>videoitem.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
